packages/admin/src/app/shared/components/people-table/people-table.component.cy.ts

```diff
@@ -0,0 +1,117 @@
+import {
+  MatCell,
+  MatCellDef,
+  MatColumnDef,
+  MatHeaderCell,
+  MatHeaderCellDef,
+  MatHeaderRow,
+  MatHeaderRowDef,
+  MatRow,
+  MatRowDef,
+  MatTable,
+  MatTableDataSource,
+} from '@angular/material/table';
+import { PeopleTableComponent } from './people-table.component';
+import { PersonModel } from '@app/shared/models/person.model';
+import { initialState } from '@app/feature/people-list/people-list.store';
+import { MatPaginator } from '@angular/material/paginator';
+import { MatIcon } from '@angular/material/icon';
+import { MatButton, MatIconButton } from '@angular/material/button';
+import { MatSortModule } from '@angular/material/sort';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { createOutputSpy } from 'cypress/angular-signals';
+import { peopleFactory } from '@app/shared/testing/factories/people';
+import { PeopleResponseType } from '@app/shared/schemas/person.schema';
+
+describe('PeopleTableComponent', () => {
+  it('can mount with correct inputs', () => {
+    const peopleList = peopleFactory(10);
+    const personResponse: PeopleResponseType = {
+      data: peopleList,
+      meta: {
+        count: peopleList.length,
+      },
+    };
+
+    const displayedColumns = [
+      'familyName',
+      'givenName',
+      'personType',
+      'tags',
+      'delete',
+    ];
+
+    cy.mount(PeopleTableComponent, {
+      imports: [
+        MatTable,
+        MatColumnDef,
+        MatRowDef,
+        MatHeaderRowDef,
+        MatCellDef,
+        MatHeaderCellDef,
+        MatRow,
+        MatCell,
+        MatHeaderCell,
+        MatHeaderRow,
+        MatPaginator,
+        MatIcon,
+        MatIconButton,
+        MatSortModule,
+        MatButton,
+        NoopAnimationsModule,
+      ],
+      autoSpyOutputs: true,
+      componentProperties: {
+        dataSource: new MatTableDataSource<PersonModel>(personResponse.data),
+        displayedColumns: displayedColumns,
+        matSortActive: initialState.sort,
+        matSortDirection: initialState.order,
+        length: peopleList.length,
+        pageSizes: [10, 25, 50],
+        pageSize: 10,
+        pageIndex: initialState.pageIndex,
+        onDeleteClicked: createOutputSpy('onDeleteClickedSpy'),
+        onPageChanged: createOutputSpy('onPageChangedClickSpy'),
+        onRowClicked: createOutputSpy('onRowClickedSpy'),
+        onSortChanged: createOutputSpy('onSortChangedSpy'),
+        onViewAccountClicked: createOutputSpy('onViewAccountClickedSpy'),
+      },
```

Comment:
What determines what you need to set here?  I see several that are are not input/outputs.  e.g. pageSizes, displayColumns, etc

Reply:
I am not following.  Page sizes and displayed columns are inputs